test/projects/projects-page.html

```diff
@@ -33,8 +33,30 @@
         projectPage.showProjectsList = true;
         MockInteractions.tap(Polymer.dom(projectPage.root).querySelector('iron-pages'));
         assert.isFalse(projectPage.showProjectsList);
+      });
+
+      test('Selecting project by title', function() {
```

Comment:
The test names do not correspond to the usual format:
`sets activeProject when project title is set`